guides/guide-for-adding-windows-node.md

```diff
@@ -114,33 +187,32 @@ kubectl apply -f sig-windows-tools/kubeadm/flannel/kube-flannel-rbac.yml
 1. Install ContainerD, wins, kubelet, and kubeadm.
 
 ```PowerShell
-git clone https://github.com/kubernetes-sigs/sig-windows-tools
-cd .\sig-windows-tools\kubeadm\scripts\
-
 # Install ContainerD
+curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/kubeadm/scripts/Install-Containerd.ps1
 .\Install-Containerd.ps1
 
 # Install wins, kubelet and kubeadm
```

Comment:
We don't need wins any more

Reply:
should remove references from the guide so not to confuse

Reply:
done